applications/DEMApplication/tests/test_post_process.py

```diff
@@ -46,16 +43,6 @@ def test_gid_printing_many_results(self):
         model = KratosMultiphysics.Model()
         auxiliary_functions_for_tests.CreateAndRunStageInSelectedNumberOfOpenMPThreads(TestPostProcessClass1, model, parameters_file_name, 1)
 
-    def tearDown(self):
```

Comment:
I would not remove these lines. They are not deleting general folders, but specific files which were created by this specific case

Reply:
fixed